domain/infosync/info.go

```diff
@@ -79,8 +79,9 @@ type ServerInfo struct {
 
 // ServerVersionInfo is the server version and git_hash.
 type ServerVersionInfo struct {
```

Comment:
How about rename the struct name and update the comment?

Reply:
I have moved the `start_timestamp` to the struct `ServerInfo`.